packages/adapter-static/test/apps/spa-entry-name/README.md

```diff
@@ -0,0 +1,38 @@
+# create-svelte
```

Comment:
can we test this feature using an existing test project? we try to minimize the number of test projects we create in order to avoid time starting / stopping the server, etc.

Reply:
I have no idea how to test in one project using two different adapter options. The structure of build dir is totally different with or without this option. May be we can use subdir to split different strategies?

```text
routes
├── default
│   ├── index.svelte
│   └── sub
│       └── page.svelte
└── strip-index
    ├── index.svelte
    └── sub
        └── page.svelte
```

```javascript
outputFileName: ({ path, is_html }) => {
	if (!is_html) return path;

	if (path.startsWith('/strip-index/')) {
		// return custom name
	}

	// return default name
}
```